src/riak_kv_qry_compiler.erl

```diff
@@ -827,12 +829,17 @@ rewrite2([#param_v1{name = [FieldName]} | T], Where1, Mod, Acc) ->
 
 %% Functions to assist with coverage chunks that redefine quanta ranges
 unwrap_cover(undefined) ->
-    {undefined, undefined};
-unwrap_cover(Cover) ->
-    {ok, {OpaqueContext, {FieldName, RangeTuple}}} =
-        riak_kv_pb_coverage:checksum_binary_to_term(Cover),
-    {riak_kv_pb_coverage:term_to_checksum_binary(OpaqueContext),
-     {FieldName, RangeTuple}}.
+    {ok, {undefined, undefined}};
+unwrap_cover(<<>>) ->
```

Comment:
I think @andytill's suggestion is good, and we shouldn't modify input like this. It wouldn't have helped in my case, anyway, as I was passing `[]` in the python client.